src/test/scala/nl.knaw.dans.easy.multideposit/actions/AddBagToDepositSpec.scala

```diff
@@ -31,6 +31,7 @@ class AddBagToDepositSpec extends TestSupportFixture with BeforeAndAfterEach {
 
   private val depositId = "dsId1"
   private val date = new DateTime(1992, 7, 30, 0, 0)
+  private val base = "ab23-bdcg"
```

Comment:
please use valid UUID's in these circumstances (not only here, but also in other test files). That's why you should use the `UUID` type instead. That one guarantees that automatically for you.